### ISSUE TITLE
make.contigs

### ISSUE THREAD

**Issue by fandemonium** [mothur/mothur#393]
mothur v1.39.5. when using function make.contigs to assemble multiple .gz files (miseq), it worked fine initially then start to return the following errors until the process gets killed. Some of the log files are 30G in size, filled with the same warnings. 

The exact command used is `make.contigs(file=stability.files, oligos=v4_primers.txt, trimoverlap=T)`

Errors are:
======
>>>>>   Processing file pair /efs/home/fyang3/monmicrobiome/workflows/SEQ29625.raw/204_S204_L001_R1_001.fastq.gz - /efs/home/fyang3/monmicrobiome/workflows/SEQ29625.raw/204_S204_L001_R2_001.fastq.gz (files 116 of 221)       <<<<<
Making contigs...
[WARNING]: Blank fasta name, ignoring read.
[WARNING]: missing sequence for , ignoring.[WARNING]: expected a name with + as a leading character, ignoring.[WARNING]: missing quality for , ignoring.[WARNING]: Blank fasta name, ignoring read.
[WARNING]: missing sequence for , ignoring.[WARNING]: expected a name with + as a leading character, ignoring.[WARNING]: missing quality for , ignoring.[WARNING]: Blank fasta name, ignoring read.
======

or error like this:
======
>>>>>   Processing file pair /efs/home/fyang3/monmicrobiome/workflows/SEQ29625.raw/156_S156_L001_R1_001.fastq.gz - /efs/home/fyang3/m
onmicrobiome/workflows/SEQ29625.raw/156_S156_L001_R2_001.fastq.gz (files 62 of 221)        <<<<<
Making contigs...
[WARNING]: Lengths do not match for sequence M02582:12:000000000-B9MBH:1:2114:13081:23840. Read 301 characters for fasta and 179 char
acters for quality scores, ignoring read.[WARNING]: Lengths do not match for sequence M02582:12:000000000-B9MBH:1:2114:13081:23840. R
ead 299 characters for fasta and 217 characters for quality scores, ignoring read.[WARNING]: Blank fasta name, ignoring read.
[WARNING]: missing sequence for , ignoring.[WARNING]: expected a name with + as a leading character, ignoring.[WARNING]: missing qual
ity for , ignoring.[WARNING]: Blank fasta name, ignoring read.
[WARNING]: missing sequence for , ignoring.[WARNING]: expected a name with + as a leading character, ignoring.[WARNING]: missing qual
ity for , ignoring.[WARNING]: Blank fasta name, ignoring read.
======

Note, the second type of error specified a sequence that had different read and quality length. But this is not true. Has anyone seen anything similar?

Thanks!
Fan

**Comment by didillysquat** [comment]
Hi there,

I'm having the same never ending stream of warnings. I started a loop to count the errors programmatically trying to see how many of the sequencing reads Mothur was having an issue with, but as Fan says, Mothur seems to be stuck in a loop somewhere and just keeps outputting the same thread. Strangely enough it only happens on some of the read pairs and not others when doing make.contigs. I have peaked into the fastq files that are causing the problems and I can't see an obvious reason for the issue.

Thanks for your help in fixing this.

Ben
